### PR TITLE
oncall  - OSD-8119  WIP

### PR DESCRIPTION
**Pull request by MitaliBhalla**

### CODE REVIEW

**Comment by T0MASD** [comment]
Good work on the PR! I was able to run oncall and returned the results, I also tried testing and it was running the test! I like your approach to use separate function for converting time :100: 
There are couple of notes:
- please run `./pdcli alerts --low` and compare the output of `./pdcli oncall` table looks differently, can you find why it's different?
- please take a look at https://github.com/openshift/pagerduty-short-circuiter/blob/main/cmd/pdcli/alerts/cmd.go and add changes for `oncall` to achieve consistency with `alerts`?

File: cmd/pdcli/oncall/cmd.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+var Cmd = &cobra.Command{
+	Use:   "oncall",
+	Short: "oncall to the PagerDuty CLI",
+	Long:  "Running the pdcli oncall command will display the current primary and secondary oncall SRE",
+	Args:  cobra.NoArgs,
+	RunE:  OnCall,
+}
+
+//TimeConversion converts timestamp into time and date
+func TimeConversion(s string) string {
```
Comment (supreeth7) [inline cmd/pdcli/oncall/cmd.go:37]: This can be a local function instead.

Comment (T0MASD) [reply cmd/pdcli/oncall/cmd.go:37]: @MitaliBhalla it doesn't look like  this function need to be exported outside oncall module
```suggestion
func timeConversion(s string) string {
```
note: references to this function will need to be updated too

**Comment by supreeth7** [comment]
I see a title war going on here. 😆 
P.S - @MitaliBhalla try changing the PR title to something like **WIP: <Titlte>** instead of "<Titlte> WIP", so that the openshift-ci bot can recognize that the PR is still work in progress and label the same.

**Comment by supreeth7** [comment]
@MitaliBhalla there's something wrong with your commits. Why're you trying to commit over my PRs?

**Comment by supreeth7** [comment]
@T0MASD can you PTAL?

**Pull request closed without merging**